nsxt/data_source_nsxt_policy_project.go

```diff
@@ -0,0 +1,83 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+	infra "github.com/vmware/vsphere-automation-sdk-go/services/nsxt/orgs"
+)
+
+const defaultOrgID = "default"
+
+func dataSourceNsxtPolicyProject() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceNsxtPolicyProjectRead,
+
+		Schema: map[string]*schema.Schema{
+			"id":           getDataSourceIDSchema(),
+			"display_name": getDataSourceDisplayNameSchema(),
+			"description":  getDataSourceDescriptionSchema(),
+			"path":         getPathSchema(),
+		},
+	}
+}
+
+func dataSourceNsxtPolicyProjectRead(d *schema.ResourceData, m interface{}) error {
+	connector := getPolicyConnector(m)
+	client := infra.NewProjectsClient(connector)
+
+	objID := d.Get("id").(string)
```

Comment:
Hi Kobi, why not use search API here, like in https://github.com/vmware/terraform-provider-nsxt/blob/master/nsxt/data_source_nsxt_policy_segment.go for example ? Are projects searchable?

Reply:
Not sure why but search doesn't seem to work for me

Reply:
I've added a comment - although it's doable with search API, it'll be quite cumbersome even after I'll add support for [multitenancy](https://github.com/vmware/terraform-provider-nsxt/compare/master...ksamoray:terraform-provider-nsxt:api-wrapper#diff-c5d8c0bd4f04ea7a2984c77a5b9c331564d1b48b518cb6a74fd54e897671a900R230) in  search API